sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Platform/EnvironmentVariableConstants.cs

```diff
@@ -65,5 +67,10 @@ internal static class EnvironmentVariableConstants
         /// When set to true, exporter will emit resources as metric telemetry.
         /// </summary>
         public const string EXPORT_RESOURCE_METRIC = "OTEL_DOTNET_AZURE_MONITOR_ENABLE_RESOURCE_METRICS";
+
+        /// <summary>
+        /// INTERNAL ONLY. Used by LiveMetrics to detect premium containers in Azure App Service.
+        /// </summary>
+        public const string WEBSITE_ISOLATION = "WEBSITE_ISOLATION";
```

Comment:
This is not a documented environment variable from Azure App Service. Ref: https://learn.microsoft.com/en-us/azure/app-service/reference-app-settings?tabs=kudu%2Cdotnet

Leave a TODO to check with App Service if we could rely on it.

Reply:
Oh weird. AI SDK has been using this for years.
I can confirm that it's there and it works for our purposes.

Adding a TODO.

Reply:
After further investigation, WEBSITE_ISOLATION is no longer needed.
This was included in AI SDK to support Premium Containers.
This feature was retired in 2022.
https://azure.github.io/AppService/2022/03/02/Migrating-your-Windows-conainer-apps-from-Premium-Container-SKU-(Preview)-to-Premium-V3-SKU.html